### ISSUE TITLE
Helix API: Port UI controllers and views

### ISSUE THREAD

**Issue by alexperovich** [dotnet/arcade#5511]

**Comment by premun** [comment]
@alexperovich would you have an ETA for this?

We are currently working on SDL 2020 (https://github.com/dotnet/core-eng/issues/10171) for Helix and there are some accessibility issues with the **helix.dot.net** website. Since we cannot roll the web out, it is impossible to fix these atm.

cc @tkapin

**Comment by alexperovich** [comment]
I am currently on FR so not this week or the next. I will probably have a PR for this by the 14th.

**Comment by premun** [comment]
Hey @alexperovich, how is it looking? Is it going to slip? Is there a way we can help with this maybe?

**Comment by alexperovich** [comment]
Yea, thats the last bit of this.

**Comment by premun** [comment]
Looks like this was completed but staging Helix website hasn't been re-deployed. Is there some last last bit of this? :)

**Comment by ChadNedzlek** [comment]
The staging site does not point to the new Helix API yet.  Now that we have the last bits, we are preparing the steps to get that changeover started.

**Comment by tkapin** [comment]
@alexperovich @ChadNedzlek - the deadline for SDL2020 #10171 - (the date has already been moved once) is end of this month, 2020-08-31. This is a blocker for https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1092128.
The PR https://dev.azure.com/dnceng/internal/_git/dotnet-helix-service/pullrequest/8964 is already merged, the last bit is to have this rolled out to production. Do you have an ETA when this is going to happen? Thanks.

FYI @Chrisboh

**Comment by ChadNedzlek** [comment]
It should be done by the end of the week.

**Comment by premun** [comment]
Thanks. One reason we are asking about this is that we didn't understand from the epic that there will be other than infrastructural changes. Example - you were changing the HTML of the Helix website - and even though the changes you made are an improvement and I think you did a good job on improving of the semantics of the page, it kind of rendered the accessibility scans, I was doing previously, invalid, since I will need to do them again.

I tried running the SF cluster locally and I was unable to run the website so that I can scan it again. Do you know if that is possible already and is there any docs on what I need to set up? The cluster deployed with 1 healthy and 1 unhealthy application (data migration, I think?), I didn't know how to access the web.

**Comment by ChadNedzlek** [comment]
It's important that you not run the Helix cluster locally. It will inject itself into the real data (SQL, EventHub, Kusto), and interfere with real events.

The site should be runnable by itself.

You can also go to dotnet-eng-helix-int.westus2.cloudapp.azure.com if you ignore the SSL exception (because it's expecting you to get there from the CNAME).

The DNS is scheduled to be switched in the next hour, and helix-int.dot.net should point there as well shortly.

**Comment by premun** [comment]
I am getting this when running the API project only:
![image](https://user-images.githubusercontent.com/7013027/90733694-3266ea80-e2cd-11ea-92e7-f5916435f177.png)

If you have some more instructions I could try?

Also seems the UI updates broke the accessibility again :(

**Comment by premun** [comment]
Btw there is no Swagger now?

**Comment by ChadNedzlek** [comment]
The swagger is there, we just didn't put a link to it in the bar. We can add that pretty easily.

That URL looks like you are running the "IIS Express" in visual studio (the name way up next to the green arrow in the toolbar).  I've never gotten that to work with aspnet core applications, and it's the default VS chooses.  You need to change it to the name of the application "Helix API", and debug that (the same as just running the program from the command line)

**Comment by premun** [comment]
@ChadNedzlek do you have an estimate on when we will roll out Service Fabric to production?

**Comment by jakubstilec** [comment]
@alexperovich @ChadNedzlek please is there an update on this? Deployment of Helix Web is the last item on Helix SDL list and deadline for it is very close - 31st Aug.

cc @ilyas1974, @tkapin

**Comment by ChadNedzlek** [comment]
We roll out on Wednesdays.  Last week there was a bug that would have broken production, so we had to abort the rollout.  The next rollout is next Wednesday, the 2nd of September

**Comment by tkapin** [comment]
@ChadNedzlek are you confident we will roll this out this week? Thanks.

**Comment by ChadNedzlek** [comment]
We're certainly going to try. But things happen.  The builds have been passing for several days, but there are some big changes that need to get rolled out (in particular, we are rolling out a new cluster, because we've automated the cluster deployment).

**Comment by jakubstilec** [comment]
@ChadNedzlek, @tkapin  I found that the issue was closed, but it hasn't been deployed. Please when is it planned to be deployed?

**Comment by ChadNedzlek** [comment]
Sorry, it got deployed but then we had to switch it back because it broke the IFrame in AZDO.
